### PR TITLE
Fix Start Singularity Debug proc

### PR DESCRIPTION
**Pull request by ghost**
- Fix emitters not working (beam wasn't created and updated properly)
- Fix field generators using the incorrect proc to start up
- Fix Singularity spawning before the field generators warmed up
- Make the radiation mix more robust, can charge all SMES on max (needs to be set manually, but shouldn't be as hard as setting up the whole Engine)

Now, you can set up the power in one click (plus four to max out the SMES cells potentially) without turning all SMES into infinites (which can fuck with engine testing)

### CODE REVIEW

File: code/modules/admin/verbs/debug.dm [hunk 1]
```diff
@@ -1078,47 +1078,55 @@ Pressure: [env.return_pressure()]"}
 
 	for(var/obj/machinery/power/emitter/E in power_machines)
 		if(E.anchored)
+			//We will reproduce ALL variables and procs that are supposed to fire, because there's no toggle proc
```
Comment (clusterfack) [inline code/modules/admin/verbs/debug.dm]: You could have made a toggle proc

File: code/modules/admin/verbs/debug.dm [hunk 2]
```diff
@@ -1078,47 +1078,55 @@ Pressure: [env.return_pressure()]"}
...
 
 	for(var/obj/machinery/field_generator/F in field_gen_list)
 		if(F.anchored)
-			F.Varedit_start = 1
-	spawn(30)
+			//The gentleman who coded this was nice enough to add a proc
+			F.turn_on()
+			F.investigation_log(I_SINGULO,"<font color='green'>activated</font> <font color='red'>via Start Singularity Debug verb.</font>")
+
+	spawn(200) //Field generators take 15 seconds to warm, so we'll give 20
```
Comment (clusterfack) [inline code/modules/admin/verbs/debug.dm]: Or, OR. Put this for loop at the end after a sleep(200)

Comment (dylanstrategie) [inline code/modules/admin/verbs/debug.dm]: Right then, sleep it is

File: code/modules/admin/verbs/debug.dm (hunk 1 above)
Comment (dylanstrategie) [inline code/modules/admin/verbs/debug.dm]: I'll see what I can do, but this'll obviously need extra testing

File: code/modules/admin/verbs/debug.dm (hunk 2 above)
Comment (dylanstrategie) [inline code/modules/admin/verbs/debug.dm]: Well then

**Pull request merged**